test/bigtest/interactors/match-profiles.js

```diff
@@ -1,16 +1,22 @@
 import {
   interactor,
   collection,
+  property,
 } from '@bigtest/interactor';
 
 import MultiColumnListInteractor from '@folio/stripes-components/lib/MultiColumnList/tests/interactor';
+import TextFieldInteractor from '@folio/stripes-components/lib/TextField/tests/interactor';
+import ButtonInteractor from '@folio/stripes-components/lib/Button/tests/interactor';
 
 import { CheckboxInteractor } from './checkbox-interactor';
 
 @interactor class MatchProfilesInteractor {
   list = new MultiColumnListInteractor('#match-profiles-list');
   selectAllCheckBox = new CheckboxInteractor('[data-test-select-all-checkbox]');
   checkBoxes = collection('[data-test-select-item]', CheckboxInteractor);
+  searchField = new TextFieldInteractor('#input-match-profiles-search');
+  searchSubmitButton = new ButtonInteractor('[data-test-search-and-sort-submit]');
+  searchSubmitButtonDisabled = property('[data-test-search-and-sort-submit]', 'disabled');

```

Comment:
I do not see much sense in that additional prop as we tend to use `searchSubmitButton.text`. So please proceed with `searchSubmitButton.disabled` use.